rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -730,6 +730,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		ui->fsrSharpeningStrength->setValue(fsr_sharpening_strength_def);
 	});
 
+	// DoubleSpinBoxes
+	m_emu_settings->EnhanceDoubleSpinBox(ui->TextureLODBias, emu_settings_type::TextureLodBias);
+	SubscribeTooltip(ui->gb_Level_of_Detail, tooltips.settings.texture_lod_bias);
+	const int texture_lod_bias_def = stoi(m_emu_settings->GetSettingDefault(emu_settings_type::TextureLodBias));
```

Comment:
```suggestion
	const double texture_lod_bias_def = stod(m_emu_settings->GetSettingDefault(emu_settings_type::TextureLodBias));
```

Reply:
Thanks yea it's not an integer. I used your other suggestion.